Sources/Atoms/PropertyWrapper/Watch.swift

```diff
@@ -28,7 +28,7 @@ public struct Watch<Node: Atom>: DynamicProperty {
     @ViewContext
     private var context
 
-    /// Creates a watch with the atom that to be watched.
+    /// Creates a watch with the atom to be watched.
```

Comment:
> Creates a watch with the atom to be watched.

May be better to revisit this.
[Options]
1. Initializer with the atom to be watched.
2. Creates a watch property wrapper with the atom to be watched.

Reply:
How about `Creates an instance with the atom to be watched.`?